src/TutorialTemplate.js

```diff
@@ -1,48 +1,71 @@
-import React from 'react';
-import styled from 'styled-components/macro';
-import meditation from './assets/image-meditation.jpg'
+import React from 'react'
+import styled from 'styled-components/macro'
+import data from './data/content.json'
 
-
-export default function TutorialTemplate (){
-    return (
-    <TutorialWrapper>
-        <img src={meditation} alt=""/>
-        <h2>5 Minuten of...</h2>
-        <p>Lorem ipsum dolor sit amet consectetur Quos accusamus libero minus ullam, beataelibero minus ullam, beatae possimus nostr</p>
-    </TutorialWrapper>
-    )   
+export default function TutorialTemplate() {
+  return (
+    <>
+      {data.map(
+        ({
+          headline,
+          title,
+          image,
+          description1,
+          description2,
+          description3,
+          id,
+        }) => (
+          <TutorialWrapper key={id}>
+            <img src={image} alt="" />
```

Comment:
In case the picture is relevant for the understanding of the exercise, I think it would be possible to add an extra key-value-pair with a description of the image to your conten.json and pass the value into the alt attribute. Example:
"alt_text": "A person sitting upright yet relaxed."

Also, I've seen that in your content.json for one object the image is missing. Maybe that's a mistake in that file and in reality that will never be the case; however, if that could be the case for some exercises, I think there should be a logic included which checks whether an image is part of the object (--> render it) or not (--> only render the rest of the map-function).

Reply:
First point - thx  for the suggestion 
Second point - I have no testing for now. ;-)